content/applications/finance/accounting/getting_started/main_concept.rst

```diff
@@ -131,8 +123,22 @@ average price, LIFO (for countries allowing it) and FIFO.
 Easy retained earnings
 ======================
 
-Retained earnings are the portion of income retained by your
-business. Odoo automatically calculates your current year earnings in
-real time so no year-end journal or rollover is required.  This is
-calculated by reporting the profit and loss balance to your balance
-sheet report automatically.
+Retained earnings are the portion of income retained by your business. Odoo automatically calculates
+your current year earnings in real-time, so no year-end journal or rollover is required. The profit
+and loss balance is automatically reported to your balance sheet report.
+
+.. seealso::
+   - :doc:`Accounting Cheat Sheet <cheat_sheet>`
+
+Fiduciaries
+===========
+
+The :guilabel:`Accounting Firms` mode can be activated by going to :menuselection:`Accounting -->
+Configuration --> Settings --> Accounting Firms mode`. When enabled:
+
+- The document's sequence becomes editable on all documents;
+- A new field :guilabel:`Total (tax incl.)` appears in order to speed up and control the encoding by
+  automating line creation with the right account and tax;
+- :guilabel:`Invoice Date` and :guilabel:`Bill Date` are automatically pre-filled when encoding a
+  transaction.
+- A :guilabel:`Quick encoding` option for Customer Invoices and Vendor bills.
```

Comment:
```suggestion
- A :guilabel:`Quick encoding` option is available for customer invoices and vendor bills.
```
- Added a verb to be consistent with the structure of the first bullet points.
- No capitals necessary.